bot/cogs/tags.py

```diff
@@ -237,14 +247,14 @@ def _command_on_cooldown(tag_name) -> bool:
 
             if isinstance(tag_data, dict):
                 log.warning(f"{ctx.author} requested the tag '{tag_name}', but it could not be found.")
-                embed.description = f"Unknown tag: **{tag_name}**"
+                embed.description = f"There wasn't a **{tag_name}** tag! Make sure you typed it correctly."
```

Comment:
I prefer the message that was already here.

Reply:
![image](https://user-images.githubusercontent.com/37349466/38466050-485ace94-3b1b-11e8-92d2-387b689ee617.png)
lemon doesn't.

Reply:
This was my review, not lemon's. He's free to comment as well.

Reply:
of course ^^